launcher/FileSystem.cpp

```diff
@@ -1002,32 +1002,32 @@ bool createShortcut(QString destination, QString target, QStringList args, QStri
 
     f.setPermissions(f.permissions() | QFileDevice::ExeOwner | QFileDevice::ExeGroup | QFileDevice::ExeOther);
 
-    return true;
+    return destination;
 #elif defined(Q_OS_WIN)
     QFileInfo targetInfo(target);
 
     if (!targetInfo.exists()) {
         qWarning() << "Target file does not exist!";
-        return false;
+        return "";
```

Comment:
It would be better to represent an error with QString() - which creates a null string where isNull is true
isEmpty is also true on a null string, so you can just keep that the same - however I still feel a null string makes the intent clearer

Reply:
Ah, that's a neat piece of info I didn't know. Thanks for the suggestion!